module_3/gift_certificates_system/repository/src/main/java/com/epam/esm/util/EsmPagination.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.util;
```

Comment:
please consider to move these kind of packages to corresponding package of layer, on example, EsmPagination is in com.epam.esm.util package and if I will use it in the service layer I will import it from com.epam.esm.util package, that doesn't tell me to which layer this class belongs. The package should be, in this case, com.epam.esm.util.repository

Reply:
Thanks, fixed.

Reply:
Thanks, fixed.